CONTRIBUTING.md

```diff
@@ -26,6 +26,18 @@ from the command line and then do a clean rebuild in Xcode.
 Changes made to dependencies in the [Cartfile](Cartfile) need to be reflected in
 [Ably.podspec](Ably.podspec) and vice-versa.
 
+### SPM and Carthage support
+
+In order to make available new public APIs to the Carthage users you must put the symlink of your new header file into the `Source/include/Ably` folder. Also symlinks for all new private headers of internal classes (that ones with "+Private.h" suffix) should go into `Source/PrivateHeaders/Ably` folder, as well as symlinks for other private headers used by the framework (from `Source/Private` folder). This is for compatibility with `SPM` project structure.
+
+You should generate symlinks from the command line - in Terminal `cd` to the symlinks folder and type:
+
+```
+ln -s HeaderName.h ../../../Source/HeaderName.h
+```
+
+Do not use symlinks generated by macOS Finder app.
```

Comment:
The Finder app doesn't generate symlinks – are you thinking of [aliases](https://en.wikipedia.org/wiki/Alias_(Mac_OS))? As described on that page, they're not the same thing.

Reply:
ee6e06dd